msbuild/Xamarin.iOS.Tasks.Core/Tasks/MTouchTaskBase.cs

```diff
@@ -561,7 +562,7 @@ protected override string GenerateCommandLineCommands ()
 						}
 					} else {
 						// other user-defined mtouch arguments
-						actualArgs.AddQuoted (StringParserService.Parse (argument, customTags));
```

Comment:
I looked at refactoring all of this gcc flag bit out but it got large and messy quickly.

Reply:
s/seperatly/separately/g